packages/codemod/src/upgrade-legacy.ts

```diff
@@ -30,19 +30,16 @@ class ExpectedError extends Error {
 const isInternalBlitzMonorepoDevelopment = fs.existsSync(
   path.join(__dirname, "../../../blitz-next"),
 )
-
+type TStep = {name: string; action: (stepIndex?: number) => Promise<void>}
```

Comment:
It's not a big deal — I'm only suggesting this because we don't prefix types with `T` in the codebase (we only do it for generic arguments sometimes)

```suggestion
type Step = {name: string; action: (stepIndex?: number) => Promise<void>}
```